docs/content/mobile/android/local-deploy.md

```diff
@@ -14,52 +16,47 @@ To connect your Android device to a local deployment, you need to prepare its co
 1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
 1. Select **Save** and close the window.
 
-## Use Unity Remote {#remote}
+## Unity Editor{#in-editor}
+1. In the Unity Editor, Select **SpatialOS** > **Local launch**. It’s done when you see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`.
```

Comment:
Probs put it's done on a new line to make it super clear.

Reply:
fixed